projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1037,6 +1037,10 @@
         opacity: .5;
     }
 
+    %igx-grid__tr--disabled {
+        opacity: .5;
+    }
+
```

Comment:
This is pointless. You already have the same styles defined a few lines above. You can reuse the placeholder selector.

Reply:
If that opacity fits your requirements, I will reuse the upper placeholder selector.